package.json

```diff
@@ -5,8 +5,8 @@
   "author": "Michael Davis",
   "bugs": "https://github.com/Contentstack-Solutions/contentstack-cli-tsgen/issues",
```

Comment:
bump package version

Reply:
@shafeeqd959 The API is anticipated to be released in the beginning of November, so I'll do the version bump before merging into development.